tox.ini

```diff
@@ -1,9 +1,9 @@
 [tox]
-envlist = flake8, py{36,37,38,39}, benchmark, benchmark-cuda{101,102}, mypy
+envlist = flake8, py{36,37,38,39}, benchmark, benchmark-cuda{101,102,110}, mypy
 
 [testenv]
 commands=
-    pytest --durations=10 --cov=libertem --cov-report=term --cov-report=html --cov-report=xml --junitxml=junit.xml {posargs:tests/}
+    pytest -s -vvv --durations=10 --cov=libertem --cov-report=term --cov-report=html --cov-report=xml --junitxml=junit.xml {posargs:tests/}
```

Comment:
Should we keep the very verbose `-s` output for a while? I added this because the build sometimes hung without having any indication how far it got and why it hung. If things stabilize a bit, we should switch this back off, as it can be very overwhelming.

Reply:
If we must... 🙈 😆  Ok, if it is temporary!